### DISCUSSION TITLE
Enable STARTTLS option for smtp email

### DISCUSSION THREAD

**Discussion by Alluuu** [directus/directus#6282] [Archive]
Hi,

When trying to use smtp.office365.com:587 for the smpt email, I get error:1408F10B:SSL routines:ssl3_get_record:wrong version number smtp. This seems to be because the email service requires starttls smtp for initiating the connection and doesn't support any other tls

Regards,

Alari

**Comment by rijkvanzanten** [comment]
You can already achieve this by setting "secure" to `false`, and `ignoreTLS` to `true`. Bit confusing, but see https://nodemailer.com/smtp/#tls-options for the exact combination of values + what it makes `nodemailer` do 👍🏻

**Reply by Alluuu** [reply]
Hi, thanks for the reply, I tried that actually, and received error STARTTLS is required or something similar.. HOWEVER, as I was ready to give up, I removed both those lines from the env file, and it miraculously started working :D
